src/utils/convertors.ts

```diff
@@ -500,6 +500,16 @@ export const escapeCsvValue = (value: string): string => {
     return `"${value.replace(/"/g, '""')}"`;
 };
 
+let csvSeparator: string = ';';
+
+export function setCsvSeperator(seperator: string): void {
```

Comment:
See above, loading the setting directly here would be much cleaner without any getter/setter.

Reply:
Problem is that I always get doing that:
![image](https://github.com/user-attachments/assets/c87e5030-0c9b-4ef6-a3b8-0fdd6677d49d)

Reply:
see above.